css/footer.css

```diff
@@ -0,0 +1,27 @@
+.footer{
+    background-color: var(--color-yellow);
+    text-align: center;
+    color: var(--defaul-color);
```

Comment:
Do check and correct the variable name of color.

Reply:
sure